internal/serialization/portable_serializer_test.go

```diff
@@ -37,7 +37,21 @@ func (*portableFactory1) Create(classID int32) serialization.Portable {
 	return nil
 }
 
-func (*portableFactory1) FactoryID() int32 {
+func (portableFactory1) FactoryID() int32 {
+	return 2
+}
+
+type portableFactory2 struct {
```

Comment:
Where do we use this?

Reply:
That's used in the morphing portable reader test.